engine/movie/gold_silver_intro.asm

```diff
@@ -272,54 +270,54 @@ GFPresents_UpdateLogoPal:
 GFPresents_Sparkle:
 ; Initialize and configure a sparkle sprite.
 
-	; run only every second frame
+; run only every second frame
 	ld d, a
 	and 1
 	ret nz
 
-	; shift over so our index is still changing by 1 each time
+; shift over so our index is still changing by 1 each time
 	ld a, d
 	srl a
 
-	; set up a new sparkle sprite
+; set up a new sparkle sprite
 	push af
 	depixel 11, 11
 	ld a, SPRITE_ANIM_INDEX_GS_INTRO_SPARKLE
 	call InitSpriteAnimStruct
 	pop af
 
-	; take the bottom 4 bits of a as an index into
-	; sparkle_vectors (16 entries)
+; take the bottom 4 bits of a as an index into
+; sparkle_vectors (16 entries)
 	and %00001111
 	ld e, a
 	ld d, 0
 	ld hl, .sparkle_vectors
 	add hl, de
 	add hl, de
 
-	; set the angle and distance for this sprite
-	; bc+$b <- hl (angle), bc+$c <- 0, bc+$d <- hl+1 (distance)
+; set the angle and distance for this sprite
+; bc+$b <- hl (angle), bc+$c <- 0, bc+$d <- hl+1 (distance)
```

Comment:
I'd remove this comment and add `; angle` and `; distance` next to the relevant `ld [hl]` lines. Or change it to:

```
; bc[SPRITEANIMSTRUCT_JUMPTABLE_INDEX] = [hl+0] (angle)
; bc[SPRITEANIMSTRUCT_0C] = 0
; bc[SPRITEANIMSTRUCT_0D] = [hl+1] (distance)
```